packages/smart-wallet/src/smartWallet.js

```diff
@@ -725,7 +954,10 @@ export const prepareSmartWallet = (baggage, shared) => {
           return this.state.updateRecorderKit.subscriber;
         },
         getPublicTopics() {
-          const { currentRecorderKit, updateRecorderKit } = this.state;
+          const {
+            state: { currentRecorderKit, updateRecorderKit },
+          } = this;
```

Comment:
this is a change in style. what motivates it? do you think it should be used generally?

Reply:
I think we do this occasionally. I did it here because those names were used a few times each.  I'll revert rather than figure out whether it's common.